helm/auto-deployment/configuration.md

```diff
@@ -0,0 +1,444 @@
+          - cd2.abc.com
+        secretName: tls-secret
+```
+
+## Auto Scaling
+
+By default autoscaling is enabled in Bold BI, to disable autoscaling please set `autoscaling.enabled=false`.
+
+```console
+autoscaling:
+  enabled: true
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+  behavior:
+    stabilizationWindowSeconds: 60
+    podsValue: 1
+    podsPeriodSeconds: 60
+    percentValue: 10
+    percentPeriodSeconds: 60
+```
+
+<br/>
+<table>
+    <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.enabled
+      </td>
+      <td>
+       By default the autoscaling will be enabled. turn this to <i>false</i> to disable the autoscaling functionality.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.targetCPUUtilizationPercentage
+      </td>
+      <td>
+       The CPU utilization is the average CPU usage of a all pods in a deployment across the last minute divided by the requested CPU of this deployment.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.targetMemoryUtilizationPercentage
+      </td>
+      <td>
+       With this metric the HPA controller will keep the average utilization of the pods in the scaling target at the value mentioned (80%). Utilization is the ratio between the current usage of resource to the requested resources of the pod.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.behavior.stabilizationWindowSeconds
+      </td>
+      <td>
+       The stabilization window is used by the autoscaling algorithm to consider the computed desired state from the past to prevent scaling. By default the value is 60 to provide a custom downscale stabilization window of 1 minute.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.behavior.podsValue<br/>
+       autoscaling.behavior.podsPeriodSeconds
+      </td>
+      <td>
+       This policy (Pods) allows 1 replicas to be scaled down in one minute.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling.behavior.percentValue<br/>
+       autoscaling.behavior.percentPeriodSeconds
+      </td>
+      <td>
+       This policy (Percent) allows at most 10% of the current replicas to be scaled down in one minute.
+      </td>
+    </tr>
+</table>
+<br/>
+
+## Bing Map Widget
+
+If you need to use **Bing Map** widget feature, enable this to `true` and API key value for `<widget_bing_map_api_key>`. By default this feature will be set to false.
+
+```console
+bingMapWidget:
+  enabled: <true / false>
+  apiKey: <widget_bing_map_api_key>
+```
+
+> **Note:** The Bing Map keys will be maintained in a secret named `bold-secret`
+
+## Custom Locale Path
+
+Localization is adapting a website into different linguistic and cultural contexts - involving much more than the simple text translation.By default, the Bold BI application is available in English(en-US) localization.

```

Comment:
Localization is adapting a website to different linguistic and cultural contexts - involving much more than simple text translation. By default, the Bold BI application is available in English(en-US) localization.
If you have generated a locale.js file, specify its path in <locale.js_file_path>.

Reply:
concern addressed